Core/StoreConstants.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  StoreConstants.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+
+import Foundation
+
+struct StoreConstants {
+
+    static let groupName = "group.com.duckduckgo.contentblocker"
```

Comment:
Nice work! This seems really generic but we have a few groups. Can we give this a specific name to avoid confusion e.g `contentBlockerGroup`?

Reply:
Renamed to ContentBlockerStoreConstants